### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by willowisp01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,16 @@
...
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* This is **an app for CS instructors that helps manage weaker students**.<br>
```
Comment (yashma-sonara) [inline README.md:7]: The updated description and ui looks good.

**Review by yashma-sonara — APPROVED** [review]

**Comment by willowisp01** [comment]
there's a merge conflict 😭😭😭 let me try and fix it

**Review by JiaXinEu — APPROVED** [review]
LGTM

**Comment by JiaXinEu** [comment]
There seems to be a conflict in this PR

**Pull request merged**